test/integration/forms/form.js

```diff
@@ -951,6 +951,52 @@ context('Patient Form', function() {
       .find('textarea[name="data[storyTime]"]');
   });
 
+  specify('form scripts and reducers', function() {
```

Comment:
We weren't actually previously testing scripts or reducers in this app.  So this simple test checks that the reducers are working and have access to the context scripts.